src/lib/managers/index.ts

```diff
@@ -2,4 +2,4 @@ export * as AdminEventManager from './AdminEventManager';
 export * as AdminStoreManager from './AdminStoreManager';
 export * as AuthManager from './AuthManager';
 export * as EventManager from './EventManager';
-export * as StorageManager from './StoreManager';
+export * as StoreManager from './StoreManager';
```

Comment:
it's interesting that you didn't have to change this name anywhere else. is it because files are directly importing from `StoreManager.ts` instead of `managers/index.ts`?

Reply:
Yep, the only other use is in the cart and it imports directly from the file

Reply:
should update cart to import from `@/lib/managers` instead